kivymd/uix/label/label.py

```diff
@@ -552,9 +552,9 @@ def open_context_menu(self, instance_label: CopyLabel) -> None:
 class MDLabel(
     DeclarativeBehavior,
     ThemableBehavior,
-    TouchBehavior,
     Label,
     MDAdaptiveWidget,
+    TouchBehavior,
```

Comment:
Won't these changes break the newly [added](https://github.com/kivymd/KivyMD/pull/1394) functionality?

Reply:
In the example code from this PR, double-clicking still causes copying.

Reply:
Since we have no tests, such changes in the order of child classes need thorough tests: `md_bg_color`, `adaptive_size/width/height` and so on should be tested.

Reply:
I'm not sure that my knowledge is enough to conduct such tests. It would be nice to somehow invite @Neizvestnyj to check if this breaks his logic. However, `TouchBehavior.__init__` performs only one task: subscribes to the `on_touch_down` and `on_touch_up` events. So, the only reason something can break is if the order of calling handlers of these events matters. But neither `Label` nor `MDAdaptiveWidget` subscribes to these events.